abel/tools/symmetry.py

```diff
@@ -54,13 +50,13 @@ def get_image_quadrants(IM, reorient=True, symmetry_axis=None,
          +--------o--------+ --(output) -> ----o----
          |  *     |     *  |               cQ2 | cQ3
          |   *    |    *   |
-         | Q2  *  | *   Q3 |          cQi == averaged combined quadrants
+         | Q2  *  | *   Q3 |          cQi == combined quadrants
          +--------+--------+                 
 
         symmetry_axis = None - individual quadrants
-        symmetry_axis = 0 (vertical) - average Q0+Q1, and Q2+Q3
-        symmetry_axis = 1 (horizontal) - average Q1+Q2, and Q0+Q3
-        symmetry_axis = (0, 1) (both) - combine and average all 4 quadrants
+        symmetry_axis = 0 (vertical) - sum Q0+Q1, and Q2+Q3
```

Comment:
The adding seems like it might result in some weird situations. For example, if somebody uses `symmetry_axis=(0)`, and then uses `quadrants=(True, True, False, True)`. In this situation, the top quadrants would be twice as intense as the bottom.